File: JAVA/Program/InputDevice.java
package Program;
public class InputDevice extends Komponen {
    private String jenisInput;
    private String inputLatency;

    public InputDevice() {
    }

    public InputDevice(String jenisInput, String inputLatency, String merk, String nama) {
        super(merk, nama);
        this.jenisInput = jenisInput;
        this.inputLatency = inputLatency;
    }

    public void setJenisInput(String jenisInput) {
        this.jenisInput = jenisInput;
    }

    public void setInputLatency(String inputLatency) {
        this.inputLatency = inputLatency;
    }

    public String getJenisInput() {
        return jenisInput;
    }

    public String getInputLatency() {
        return inputLatency;
    }
}

File: PYTHON/Program/InputDevice.py
from Komponen import Komponen

class InputDevice(Komponen):
    def __init__(self, jenisInput='', inputLatency='', merk='', nama=''):
        super().__init__(merk, nama)
        self.jenisInput = jenisInput
        self.inputLatency = inputLatency

    def setJenisInput(self, jenisInput):
        self.jenisInput = jenisInput

    def setInputLatency(self, inputLatency):
        self.inputLatency = inputLatency

    def getJenisInput(self):
        return self.jenisInput

    def getInputLatency(self):
        return self.inputLatency
